enterprise/frontend/src/embedding-sdk/dev.md

```diff
@@ -52,14 +52,15 @@ CYPRESS_ALL_FEATURES_TOKEN=  ${usual token from password manager}
 CYPRESS_NO_FEATURES_TOKEN=  ${usual token from password manager}
 ```
 
-And then run:
+Cypress will use the built package, so you'll have to build the sdk first (see above).
```

Comment:
Moved this part before the `run` command, as the dev probably wants to do the build first. When we originally wrote the docs the run command included the build so it wasn't needed